src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -918,7 +918,10 @@ void InferRequest::prepare_input(const cldnn::primitive_id& inputName, Blob::Ptr
             _deviceInputs[inputName] = reinterpret_device_blob(_deviceInputs[inputName], inputBlob->getTensorDesc());
         }
     } else if (input_layout.is_static() && !is_dev_input && can_use_usm) {
-        allocate_dev_mem_if_needed(_deviceInputs, inputBlob, inputName, input_layout);
+        bool need_lockable_mem = false;
+        if (prec == Precision::I16 || prec == Precision::U16 || prec == Precision::FP64 || prec == Precision::U64 || prec == Precision::U32)
```

Comment:
nit: we'd better have a common function to check unsupported input precision to ensure that this condition is aligned with copy logic

Reply:
I'd suggest to cherry-pick this commit as is to release branch and improve code for master, so we minimize risk to break something in release.

Reply:
I created a new PR to release. https://github.com/openvinotoolkit/openvino/pull/14532

Reply:
I added a function `conv_to_supported_prec` to convert original precision to supported precision. Thank you.